media/com_templates/js/admin-template-compare.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+(function() {
+
+	document.addEventListener('DOMContentLoaded', function() {
```

Comment:
Can you make an es6version of this file?

Reply:
From a quick glance it looks ok, but the nesting blocks should be reduced. There is a good article [here](https://en.wikibooks.org/wiki/Computer_Programming/Coding_Style/Minimize_nesting) about possible solutions.

Reply:
And @laoneo can we use local storage to make the button more adaptive while page refresh.

Reply:
Yes sure, we use local storage in MM as well.